presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -254,6 +254,10 @@ Property Name                             Description
 ``format_version``                         Optionally specifies the format version of the Iceberg
                                            specification to use for new tables, either ``1`` or ``2``.
                                            Defaults to ``1``.
+
+``delete_mode``                            Optionally specifies the write delete mode of the Iceberg
+                                           specification to use for new tables, either ``copy-on-write``
+                                           or ``merge-on-read``. Defaults to ``merge-on-read``.
```

Comment:
As a "nice to have but not necessary in this PR", would you consider adding a Default column for this table? If not no worries, we could do it in future as a separate docs-only PR. `partitioning` and `location` don't appear to have default values which makes sense they wouldn't, so their entry in the column could be empty, or `none`.

Reply:
New column looks good in new build, thanks for adding it!